src/gui/dialogs/DialogTimed.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file DialogTimed.hpp
+ * @author Radek Vana
+ * @brief Base of dialog which opens after period without knob event and closes on knob event
+ * @date 2021-03-05
+ */
+#pragma once
+
+#include "IDialog.hpp"
+
+class TimedDialog : public AddSuperWindow<IDialog> {
```

Comment:
Shouldn't this be „DialogTimed“ to have the same name as the file (and as other dialogs)?